project.clj

```diff
@@ -17,71 +17,80 @@
    [org.clojure/tools.reader "0.10.0"]
    [com.taoensso/timbre      "4.3.1"]]
 
+  :cljx
```

Comment:
Could you move this section back to where it was in the previous version? It makes the diffs easier to read and understand.